app/controllers/api/v3/bikes.rb

```diff
@@ -0,0 +1,176 @@
+module API
+  module V3
+    class Bikes < API::Base
```

Comment:
I can't figure out the proper way to inherit from the previous API version. Something like `class Bikes < API::V2::Bikes` doesn't pull in any of the methods or helpers, so I replicated them all here so I wouldn't get blocked.